kubejs/assets/naturesaura/lang/en_us.json

```diff
@@ -0,0 +1,223 @@
+    "advancement.naturesaura.conversion_catalyst": "Not So Equivalent Exchange",
+    "advancement.naturesaura.conversion_catalyst.desc": "Create a Conversion Catalyst for easy material conversion",
+    "advancement.naturesaura.infused_tools": "Gear Up Intensifies",
+    "advancement.naturesaura.infused_tools.desc": "Create an Infused Iron Pickaxe and Blade",
+    "advancement.naturesaura.aura_bottle_overworld": "A Taste of Sunlight",
+    "advancement.naturesaura.aura_bottle_overworld.desc": "Collect Aura using a Bottle in the Overworld",
+    "advancement.naturesaura.aura_bottle_nether": "Spooky Scary Skeletons",
+    "advancement.naturesaura.aura_bottle_nether.desc": "Collect Aura using a Bottle in the Nether",
+    "advancement.naturesaura.aura_bottle_end": "Breathy Surroundings",
+    "advancement.naturesaura.aura_bottle_end.desc": "Collect Aura using a Bottle in the End",
+    "advancement.naturesaura.offering": "Yo God, Ya Want This?",
+    "advancement.naturesaura.offering.desc": "Create an Offering Table for the Offering to the Gods",
+    "advancement.naturesaura.sky_ingot": "Sturdy and Light",
+    "advancement.naturesaura.sky_ingot.desc": "Create an Ingot of the Skies using the Offering to the Gods",
+    "advancement.naturesaura.aura_cache": "Ca-ching",
+    "advancement.naturesaura.aura_cache.desc": "Create an Aura Cache to store Aura in your inventory",
+    "advancement.naturesaura.positive_imbalance": "Luscious Surroundings",
+    "advancement.naturesaura.positive_imbalance.desc": "Be in an area with enough Aura to cause positive imbalance effects",
+    "advancement.naturesaura.negative_imbalance": "Deadly Day",
+    "advancement.naturesaura.negative_imbalance.desc": "Drain enough Aura to cause negative imbalance effects",
+    "advancement.naturesaura.end_flower": "Blue Lotus",
+    "advancement.naturesaura.end_flower.desc": "Kill the Ender Dragon to make a Rose of Oblivion grow",
+    "advancement.naturesaura.eye": "Suddenly I See",
+    "advancement.naturesaura.eye.desc": "Create an Environmental Eye to see the Aura in the area",
+    "advancement.naturesaura.eye_improved": "Suddenly I See a Lot Better",
+    "advancement.naturesaura.eye_improved.desc": "Create an Environmental Ocular to see Aura imbalance effects",
+    "advancement.naturesaura.range_visualizer": "I Spy With my Little Eye",
+    "advancement.naturesaura.range_visualizer.desc": "Create a Mystical Magnifier to see the range of your devices",
+    "command.naturesaura.aura.usage": "/naaura store|drain <amount> [range] OR /naaura reset <range>",
+    "effect.naturesaura.breathless": "Breathless",
+    "entity.naturesaura.effect_inhibitor": "Effect Powder",
+    "entity.naturesaura.mover_cart": "Aura Attraction Cart",
+    "enchantment.naturesaura.aura_mending": "Nature's Mend",
+    "enchantment.naturesaura.aura_mending.desc": "Repairs tools using Aura",
+    "naturesaura:aura_mending.enchant.desc": "Heals the player using Aura"
+}
```

Comment:
Erhm... Why is this lang file here? 😄

Reply:
Possibly controversial. 

Since I changed the base material for Infused Iron from Iron to Sunmetal, I thought it best to re-name the item as well.

Reply:
Ahh okay - Please put the changed lang entries in `kubejs\assets\enigmatica\lang\en_us.json` instead of overwriting the original :)

Reply:
Love that renaming btw, Sunmetal sounds awesome.

Reply:
Actually, now that I think about it... I wonder if I can -only- override those two names somehow without re-writing the whole damn lang file

Reply:
You can, simply only change those two but in our own langfile so the original is kept.

Reply:
page wasn't refreshing >.< Will get that in now

Reply:
There we go. All nice and shiny